### PR TITLE
Updating Documentation

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,253 +9,54 @@ Lumen + AngularJS project seed with Docker
...
+* [Clarity](https://github.com/tobi/clarity)
+* [Mailcatcher](http://mailcatcher.me/)
+* [Beanstalkd](https://github.com/kr/beanstalkd)
+* [Redis](http://redis.io/)
+
+## Setup Instructions
+Spira has a yeoman generator at https://github.com/spira/generator-spira which is the easiest way to set up the project. To set up the project, do the following:
+
+1. Install [npm](https://www.npmjs.com/).
+2. Ensure that you have added an authorise token from your github account. Part way through the composer install section you may get rate limited by github. See instructions [here](https://help.github.com/articles/creating-an-access-token-for-command-line-use/).
```
Comment (zakhenry) [inline README.md:37]: Is this needed? numbering skips from 2 to 5

**Comment by zakhenry** [comment]
@jeremysik you will need to remove the tsd reinstall from the composer.json postinstall script.

File: README.md (hunk 1 above)
Comment (ghost) [inline README.md:37]: Fixed.

**Comment by ghost** [comment]
@xiphiaz fixed. Removed tsd reinstall from post install script.

**Pull request merged**